src/Route.php

```diff
@@ -95,7 +95,7 @@ public function __construct(Server $server, IlluminateRoute $route)
         }
 
         $this->uri = str_replace(
-          $this->server->baseUri(),
+          $this->server->url(),
```

Comment:
The doubled uri part comes from here. The public `url` method is generally a better idea, because it doesn't require to make the `baseUri` public. 
But this break the way how the server prefix is removed. A way could be to convert the resource URI to an URL and then do the replacing.
To clarify the bug

| | Server | Resource |
|---|---|---|
|Original|`/api/v1` | `/api/v1/resource` | 
| PR| `http://localhost/api/v1` | `/api/v1/resource`  |
| Fixed | `http://localhost/api/v1` |  `http://localhost/api/v1/resource` |